interface/forms/fee_sheet/review/fee_sheet_classes.php

```diff
@@ -74,7 +76,16 @@ class procedure extends code_info
     function __construct($c, $ct, $desc, $fee, $justify, $modifiers, $units, $mod_size, $selected = true)
     {
         parent::__construct($c, $ct, $desc, $selected);
-        $this->fee=$fee;
+
+        $currentPrice = sqlQuery("SELECT codes.id, prices.pr_price FROM `codes` LEFT JOIN prices ON prices.pr_id = codes.id WHERE codes.code = ?", array($c));
+
+        if($currentPrice['pr_price'] != $fee){
```

Comment:
When dealing with real numbers it is best to use strict comparison and not compare strings. So i'd cast and compare.
`if((0.00 + $currentPrice['pr_price']) !== (0.00 + $fee))` especially if dealing with a previous user input where string formats may not be the same.

Reply:
Ok, I learned something here. I will go look it up and add it to my repertoire